lib/publishing_api_payload.rb

```diff
@@ -22,9 +22,9 @@ def payload
       links: links,
       access_limited: access_limited,
       auth_bypass_ids: auth_bypass_ids,
-      public_updated_at: history.public_updated_at,
+      public_updated_at: history.public_updated_at.rfc3339,
```

Comment:
We shouldn't need this by the way following: https://github.com/alphagov/content-publisher/pull/1887